sapl/norma/forms.py

```diff
@@ -89,6 +91,16 @@ def filter_ementa(self, queryset, name, value):
             q &= Q(ementa__icontains=t)
 
         return queryset.filter(q)
+    
+    def filter_vigencia(self, queryset, name, value):
+        data_atual = timezone.now()
+        if value: # É vigente
+            queryset = queryset.filter((Q(data_vigencia__gt=data_atual) or Q(data_vigencia__isnull=True)) and Q(data__lte=data_atual))
```

Comment:
Eu não sabia que ele aceitava `and` and `or`. Sempre usei `|` (or) e `&` (and).

Reply:
Também funciona, mas substitui pra manter o padrão.